### DISCUSSION TITLE
Run Terminal at startup minimized

### DISCUSSION THREAD

**Discussion by christianavi** [microsoft/terminal#12974] [General]
Is there any way to run Terminal at startup minimized?

I don't know if there is any use case for this, but as far as I am concerned. The reason it would be useful is because I realized the quake terminal only opens with Win + ` **while** Terminal app is open.
![image](https://user-images.githubusercontent.com/43099141/164995369-b4893ba2-ed7e-4267-8897-a270e3a9af12.png)
Now this is silly because each time, I have to open a Terminal window, open a quake terminal and close the first window I opened while the Terminal window remains. I tried to set it to launch at startup, but there is no option to just launch it minimized so I always have to do the same every time.

**Comment by zadjii-msft** [comment]
What you're basically looking for is tracked in /dup #9996. There's also an issue for starting the Terminal minimized, but I think #9996 is really the crux of "I want to immediately be able to hit <kbd>win+`</kbd> and have it just work"

**Reply by christianavi** [reply]
Wow. I did not realize there were already so much looking for the same thing. Goes to say how convenient it would really be to have it. Thanks!
